### DISCUSSION TITLE
Exporting to mermaid/other markdown supported format

### DISCUSSION THREAD

**Discussion by qwertynik** [tone-row/flowchart-fun#250] [Q&A] [answered]
Hello,

Thanks for building this. Looks powerful in terms of making documentation more descriptive without significant context switches.

Is it possible to convert the diagram to [mermaid](https://github.com/mermaid-js/mermaid), or, other markdown supported format?

Thanks!

**Comment by rob-gordon** [comment, marked as answer]
My tweet from a couple days ago: https://twitter.com/tone_row_/status/1493328350915477511?s=20&t=hfyFTGTVKY_Hoy7UKmSweQ

In short, yes, this is on the roadmap =) It should be ready within a matter of weeks 🚀

**Reply by qwertynik** [reply]
This should make the tool a part of the development workflow for a much larger group of users 🚀
Looking forward to it @rob-gordon

**Comment by rob-gordon** [comment]
@qwertynik Export to Mermaid is available now

<img width="1552" alt="Screen Shot 2022-02-27 at 11 38 43 PM" src="https://user-images.githubusercontent.com/159949/155925033-07d34310-3941-4738-9c14-6d8787ac9c16.png">

**Reply by qwertynik** [reply]
That's great @rob-gordon 🎉🎆
